examples/map/image_bright_regions_gallery_example.py

```diff
@@ -60,38 +40,33 @@
 ##############################################################################
 # Only the brightest pixels remain in the image.
 # However, these areas are artificially broken up into small regions.
-# Estimating the number of significant hot regions will be difficult.
 # We can solve this by applying some smoothing to the image data.
 # Here we apply a 2D Gaussian smoothing function to the data.
-
-data2 = ndimage.gaussian_filter(aiamap.data * ~mask, 16)
+data2 = ndimage.gaussian_filter(aiamap.data * ~mask, 14)
 
 ##############################################################################
 # The issue with the filtering is that it create pixels where the values are
 # small (<100), so when we go on later to label this array,
 # we get one large region which encompasses the entire array.
 # If you want to see, just remove this line.
-
 data2[data2 < 100] = 0
 
 ##############################################################################
 # Now we will make a second SunPy map with this smoothed data.
-
 aiamap2 = sunpy.map.Map(data2, aiamap.meta)
 
 ##############################################################################
 # The function `label` from the `scipy.ndimage` module, counts the number of
 # contiguous regions in an image.
-
 labels, n = ndimage.label(aiamap2.data)
 
 ##############################################################################
 # Finally, we plot the smoothed bright image data, along with the estimate of
 # the number of distinct regions. We can see that approximately 6 distinct hot
 # regions are present above the 5% of the maximum level.
-
 plt.figure()
-aiamap2.plot()
+ax = plt.subplot(projection=aiamap)
+aiamap_orig.plot()
```

Comment:
this should just be `aiamap`

Reply:
I changed it on purpose. Showing the original map with the contours is more illustrative than showing the blurred out map.